plugins/importexport/native/filter/NativeXmlIssueFilter.inc.php

```diff
@@ -321,6 +317,40 @@ function parseIssueCover($node, $issue) {
 		}
 	}
 
+	/**
+	 * Parse out the issue identification and store it in an issue.
+	 * @param $node DOMElement
+	 * @param $issue Issue
+	 */
+	function parseIssueIdentification($node, $issue) {
```

Comment:
These two are different, one assigns the values directly to an issue, the other uses them to search after the issue in the DB. But I could eventually just extract the part that parse the xml and then return an array of all the values (volume, number, titles) if this would be better/cleaner solution...